skglm/utils/data.py

```diff
@@ -252,3 +253,22 @@ def _alpha_max_group_lasso(X, y, grp_indices, grp_ptr, weights):
             norm(X[:, grp_g_indices].T @ y) / (n_samples * weights[g])
         )
     return alpha_max
+
+
+def generate_GraphicalLasso_data(n_samples, n_features):
```

Comment:
no CamelCase in function names ; we use `make_xxx\data` syntax. Maybe `make_covariance_data` ?